src/static/js/earn.js

```diff
@@ -465,504 +2164,287 @@ async function main() {
+    poolShareDisplay_1_pgl = ret_1[1]
+    stakeDisplay_1 = ret_1[2]
+    totalPoolPGL_1 = ret_1[3]
+  }
+
+  let poolShareDisplay_2, poolShareDisplay_2_pgl, stakeDisplay_2, totalPoolPGL_2
+  if (stakedPool2.amount / 1e18 > 0) {
+    let ret_2 = await calculateShare(null, SNOB_AVAX_ADDR, stakedPool2.amount / 1e18, 1e18, userPool2Percent)
+    poolShareDisplay_2 = ret_2[0]
+    poolShareDisplay_2_pgl = ret_2[1]
+    stakeDisplay_2 = ret_2[2]
+    totalPoolPGL_2 = ret_2[3]
+  }
+
+  //SNOWGLOBE_PNG_ADDR
+  const snowglobeContract_3 = new ethers.Contract(SNOWGLOBE_PNG_ADDR, SNOWGLOBE_ABI, signer)
+  let poolShareDisplay_3, poolShareDisplay_3_pgl, stakeDisplay_3, totalPoolPGL_3
+  if (stakedPool3.amount / 1e18 > 0) {
+    let ret_3 = await calculateShare(
+      snowglobeContract_3,
+      PNG_AVAX_ADDR,
+      stakedPool3.amount / 1e18,
+      1e18,
+      userPool3Percent
+    )
+    poolShareDisplay_3 = ret_3[0]
+    poolShareDisplay_3_pgl = ret_3[1]
+    stakeDisplay_3 = ret_3[2]
+    totalPoolPGL_3 = ret_3[3]
+  }
+
+  //SNOWGLOBE_ETH_ADDR
+  const snowglobeContract_4 = new ethers.Contract(SNOWGLOBE_ETH_ADDR, SNOWGLOBE_ABI, signer)
+  let poolShareDisplay_4, poolShareDisplay_4_pgl, stakeDisplay_4, totalPoolPGL_4
+  if (stakedPool4.amount / 1e18 > 0) {
+    let ret_4 = await calculateShare(
+      snowglobeContract_4,
+      ETH_AVAX_ADDR,
+      stakedPool4.amount / 1e18,
+      1e18,
+      userPool4Percent
+    )
+    poolShareDisplay_4 = ret_4[0]
+    poolShareDisplay_4_pgl = ret_4[1]
+    stakeDisplay_4 = ret_4[2]
+    totalPoolPGL_4 = ret_4[3]
+  }
+
+  //SNOWGLOBE_USDT_ADDR
+  const snowglobeContract_5 = new ethers.Contract(SNOWGLOBE_USDT_ADDR, SNOWGLOBE_ABI, signer)
+  let poolShareDisplay_5, poolShareDisplay_5_pgl, stakeDisplay_5, totalPoolPGL_5
+  if (stakedPool5.amount / 1e18 > 0) {
+    let ret_5 = await calculateShare(
+      snowglobeContract_5,
+      USDT_AVAX_ADDR,
+      stakedPool5.amount / 1e18,
+      1e6,
+      userPool5Percent
+    )
+    poolShareDisplay_5 = ret_5[0]
+    poolShareDisplay_5_pgl = ret_5[1]
+    stakeDisplay_5 = ret_5[2]
+    totalPoolPGL_5 = ret_5[3]
+  }
+
+  //SNOWGLOBE_LINK_ADDR
+  const snowglobeContract_6 = new ethers.Contract(SNOWGLOBE_LINK_ADDR, SNOWGLOBE_ABI, signer)
+  let poolShareDisplay_6, poolShareDisplay_6_pgl, stakeDisplay_6, totalPoolPGL_6
+  if (stakedPool6.amount / 1e18 > 0) {
+    let ret_6 = await calculateShare(
+      snowglobeContract_6,
+      LINK_AVAX_ADDR,
+      stakedPool6.amount / 1e18,
+      1e18,
+      userPool6Percent
+    )
+    poolShareDisplay_6 = ret_6[0]
+    poolShareDisplay_6_pgl = ret_6[1]
+    stakeDisplay_6 = ret_6[2]
+    totalPoolPGL_6 = ret_6[3]
   }
-  function poolS3D(options) {
-    if (options.icequeen_apr) {
 
-      var eDayAPR = `${options.icequeen_apr.toFixed(2)}`;
-      var eWeekAPR = `${(options.icequeen_apr * 7).toFixed(2)}`;
-      var eYearAPR = `${(options.icequeen_apr * 365).toFixed(2)}`;
+  function pool(options) {
```

Comment:
This is 2 different function, **pool** and ** poolS3D**. The pool function start from line **609** of the "red" column